scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
More generic issue. The build is failing, because Maven can't use this XSD anymore to generate classes.
It looks that something is wrong in the parts
<xs:annotation>
        <xs:documentation xml:lang="en">
             .....
        </xs:documentation>
</xs:annotation>